run/shared/settings/geosfp/advection_met/geosfp.raw_3hr_0.25x0.625_PS_SPHU.txt

```diff
@@ -2,7 +2,7 @@ RUNDIR_IMPORT_MASS_FLUX_FROM_EXTDATA=.false.
 RUNDIR_USE_TOTAL_AIR_PRESSURE_IN_ADVECTION=0
 
 RUNDIR_MET_EXTDATA_PRIMARY_EXPORTS_FOR_ADVECTION="""
-UA;VA     m_s-1         N Y F0;013000 none  none U;V        ./MetDir/Y%y4/M%m2/D%d2/GEOS.fp.asm.tavg3_3d_asm_Nv.%y4%m2%d2_%h2%n2.V01.nc4 2014-02-11T01:30:00P03:00
+# --- Surface pressure and specific humidity, 3-hr instantaneous ---
```

Comment:
@lizziel; Question, is this `2014-02-11T00:00:00P03:00` the reference time for the data?